app/models/app_data.rb

```diff
@@ -0,0 +1,30 @@
+class AppData
+  def self.data
+    YAML.load_file(Rails.root.join('lib', 'app_data.yml')).deep_symbolize_keys
```

Comment:
Dont understand YAML stuff. Will it make a real schema so we can update this info or it can only send this fixture out for now?

Reply:
For now only the fixture. We will add updates later